frontend/dialob-composer-material/src/defaults/itemConfig.ts

```diff
@@ -6,145 +6,148 @@ import { BlurLinear, CalendarMonth, CheckBox, Circle, CropSquare, ErrorOutline,
 
 
 interface ItemConfig {
-  defaultIcon: React.ComponentType<SvgIconProps>,
-  items: {
-    matcher: (item: DialobItem) => boolean,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    component: React.FC<{ item: DialobItem, props?: any }>,
-    props: {
-      icon: React.ComponentType<SvgIconProps>,
-      placeholder: string,
-      treeCollapsible?: boolean
-    }
-  }[]
+	defaultIcon: React.ComponentType<SvgIconProps>,
+	items: {
+		matcher: (item: DialobItem) => boolean,
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		component: React.FC<{ item: DialobItem, props?: any }>,
+		props: {
+			icon: React.ComponentType<SvgIconProps>,
+			placeholder: string,
+			treeCollapsible?: boolean,
+			style?: 'normal' | 'success' | 'error' | 'info' | 'warning'
```

Comment:
We can't define the props here as TS types as it needs to remain freely configurable by customization implementations. Not a showstopper here but please discuss with @admirkrilasevic how it would be best to resolve.